CONTRIBUTING.md

```diff
@@ -0,0 +1,324 @@
+# Contributing Guide
+
+Thank you for considering contributing to SG-NLP! We believe that SG-NLP's future is closely tied with the community and community contributions will help SG-NLP to grow faster.
+
+## How can I contribute?
+
+- [Add a new model to the `sgnlp` package](#adding-a-new-model-to-sgnlp)
+- Add a new demo for an existing model to the SG-NLP demo website - refer to the contributing guide for the NLP Hub demo.
+- [Submit bug fixes](#submitting-bug-fixes)
+- [Add documentation](#adding-documentation)
+
+## Adding a new model to `sgnlp`
+
+### Pre-requisites
+
+1. Fork the `sgnlp` repository.
+2. Create a Python virtual environment (verison >= 3.8)
+3. Install the packages in `requirements_dev.txt` at the root of the repository using `pip install -r requirements_dev.txt`. This will install `black`, `flake8`, and `pytest` which we use for code formatting and testing.
```

Comment:
Actually this is missing the local installation for the dependencies of sgnlp.
It's not this card that should do it, but the issue is how to maintain one source of truth between setup.py and requirements.txt locally.